docker-compose.yaml

```diff
@@ -2,23 +2,27 @@ services:
   server:
     build: 
       context: ./src/server
+      dockerfile: ./docker/Dockerfile
+      target: local
     container_name: federated-collection-discovery-server
     ports:
       - "8000:8000"
     volumes:
-      - ./src/server/app:/app/app
+      - ./src/server/app:/work/app
     environment:
       - PYTHONUNBUFFERED=1
       - FEDERATED_STAC_API_URLS=https://stac.maap-project.org/,https://openveda.cloud/api/stac/,https://catalogue.dataspace.copernicus.eu/stac
       # - FEDERATED_CMR_URLS=https://cmr.earthdata.nasa.gov/search/
-
   client:
-    build: ./src/client
+    build: 
+      context: ./src/client
+      dockerfile: ./docker/Dockerfile.dev
```

Comment:
I was imagining that there would be a separate file for some kind of production image, but now that I think about it some more it is more likely that we would build the application and write it to a bucket rather than serve it on a web server.